packages/preferences-controller/CHANGELOG.md

```diff
@@ -6,6 +6,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [6.0.0]
+### Added
+- Added `getDefaultPreferencesState` function ([#3736](https://github.com/MetaMask/core/pull/3736))
+
+### Changed
+- **BREAKING** Clean up types ([#3712](https://github.com/MetaMask/core/pull/3712))
```

Comment:
Perhaps we could omit this package from this release?

Since this is a breaking change, typically we'd want to release everything that depends on this as well. But that'd be a pain, and there is another pending PR that also includes a breaking change to this package. Perhaps easier to save this for the next release

Reply:
Hmm let me try.   Since https://github.com/MetaMask/core/pull/3712 also modified assets-controllers, I'm not sure how i'td work to release assets-controllers without preferences-controller too?

Reply:
Oh that's a good point, sorry I hadn't realized that. Hmm.

Reply:
OK, maybe we include it then. I've just merged the other breaking change to this package that was pending: https://github.com/MetaMask/core/pull/3713